packages/story-editor/src/components/floatingMenu/elements/shared/useVideoTranscoding.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useLocalMedia } from '../../../../app';
+import useVideoTrim from '../../../videoTrim/useVideoTrim';
+import useFFmpeg from '../../../../app/media/utils/useFFmpeg';
+import useProperties from './useProperties';
+
+function useVideoTranscoding() {
+  const { resource } = useProperties(['resource']);
+  const { isTranscodingEnabled } = useFFmpeg();
+  const {
+    muteExistingVideo,
+    isResourceTrimming,
```

Comment:
@timarney, @spacedmonkey: I'm getting [an error for this one in karma](https://github.com/GoogleForCreators/web-stories-wp/runs/5421645159?check_suite_focus=true#step:7:837)? Can this not be run in karma or? Can the function at least exist even if not doing the right thing?

Reply:
See https://github.com/GoogleForCreators/web-stories-wp/pull/10784/files#r818585757 and https://github.com/GoogleForCreators/web-stories-wp/pull/10712 for context.